lib/stdlib/src/erl_stdlib_errors.erl

```diff
@@ -633,6 +635,11 @@ check_io_arguments([Type|TypeT], [Arg|ArgT], No) ->
              check_io_arguments(TypeT, ArgT, No+1)]
     end.
 
+format_json_error(_F, _As, #{position := Position}) ->
+    [{general, io_lib:format("Occured at position ~w", [Position])}];
```

Comment:
"Occurred" is misspelled.

Instead of simply fixing the misspelling, I suggest that you generate a clearer message:

```suggestion
format_json_error({invalid_byte, Byte}, #{position := Position}) ->
    S = io_lib:format("invalid byte ~w at position ~w", [Byte, Position]),
    [{general, S}];
```

(You will have to update the call to `format_json_error`, too.)

This will result in some redundancy, but overall I think it's clearer:

```
1> json:decode(~'["valid string", not_valid]').
** exception error: {invalid_byte,111}
     in function  json:invalid_byte/2 (json.erl, line 541)
        *** invalid byte 111 at position 18
     in call from json:decode/1 (json.erl, line 879)
```

Reply:
Changed it to include the character as well, might help if it is an ASCII character.

1> json:decode(<<"[\"valid string\", #not_valid">>).
** exception error: {invalid_byte,35}
     in function  json:invalid_byte/2 (json.erl, line 541)
        *** Invalid byte 35 '#' at byte position 17
     in call from json:decode/1 (json.erl, line 879)
2>

Reply:
What if it is a control character?

Reply:
Looks ok to me
```
11> json:decode(<<"[\"valid string\"", 8>>).
** exception error: {invalid_byte,8}
     in function  json:invalid_byte/2 (json.erl, line 541)
        *** Invalid byte 8 '^(' at byte position 15
     in call from json:decode/1 (json.erl, line 879)
```

Reply:
In Jason we print the byte in hex notation, and as a string if it happens to be a printable character, see https://github.com/michalmuskala/jason/blob/984bc078eb4b2084104751c7f1c5290b8338e06b/lib/decoder.ex#L12-L23

Reply:
Good point and the byte is already included as a decimal integer in the original error reason.